ci/default.rb

```diff
@@ -40,11 +40,24 @@
     task before_script: ['ci:common:before_script']
 
     task :lint do
-      sh %(find . -name '*.py' -not \\( -path '*.cache*' -or -path '*embedded*' -or -path '*venv*' \\) | xargs -n 1 pylint --rcfile=./.pylintrc)
+      output = %x[git status --porcelain]
+      paths = output.lines.collect{ |p|
+        p.strip.split ' '
+      }.select{ |stat_path|
+        status = stat_path.first
+        path = stat_path.last
+        path.end_with? ".py" and status != 'D'
```

Comment:
It's a good idea but I think it should end up in a different rake task, people have different workflows. For instance in my case `git status --porcelain` would probably return nothing in 99% cases because my changes are always committed in a WIP commit on a branch and I carry non-committed files around.
Can you do maybe a `rake lint:changed` with this behavior and keep the `rake:lint`?